js/views/homeVw.js

```diff
@@ -166,7 +166,8 @@ module.exports = Backbone.View.extend({
 
```

Comment:
Home view should also check each itemShortVw that comes in from the sockets, and not show that item if it is from a blocked user.

When you block a user, it should remove any items that are currently visible. 

That also means we don't really need code for an unblock button on itemShort, since once a user is blocked, their items should no longer be visible unless you use the address bar to go directly to their user profile. 

When you do that, I'm not sure if we should show the user profile page, or just show a screen saying they are blocked with an option to unblock them.